### ISSUE TITLE
react-css-themr should be in peerDependencies

### ISSUE THREAD

**Issue by javaguru-ch** [react-toolbox/react-toolbox#1378]
I think the module `react-css-themr` should be listed in the `peerDependencies` of the `package.json` instead of the `dependencies`. Otherwise, if I use both modules (`react-cssthemr` and `react-toolbox`) in a project, I'll end up having the `react-css-themr` module twice in my bundle.

**Comment by KhushilMistry** [comment]
Is this issue still valid ?

**Comment by luchillo17** [comment]
Yes, it's still valid, i still see the dependency, haven't been moved to peer ones yet, although the library does expose the `ThemeProvider` so maybe just use that one?

I would say the issue is that the docs doesn't tell you you can import the provider directly from `react-toolbox`.

**Comment by traksaw** [comment]
Is anyone still working on this? If not, I would like to own this.

**Comment by luchillo17** [comment]
Bruh the project is abandoned, the last commit was 6 years ago, It's kind of weird it has yet to be archived.
